src/Framework/Framework/Binding/Expressions/ValueBindingExpression.cs

```diff
@@ -49,6 +53,8 @@ private protected override void StoreProperty(object p)
         {
             if (type == typeof(KnockoutExpressionBindingProperty))
                 return knockoutExpressions.GetValue(this).GetValue(errorMode, this, type);
+            if (type == typeof(ReferencedViewModelPropertiesBindingProperty))
+                return referencedPropertyExpressions.GetValue(this).GetValue(errorMode, this, type);
```

Comment:
It's not wrong, but this storing properties in fields instead of the ConcurrentDictionary is just an optimization for the most used binding properties. This one is probably not going to be used that heavily (or?), but also not super rarely, so whatever. Just noting it so you know that you don't have to do this if you find yourself adding another binding property in the future.